yaptide/routes.py

```diff
@@ -37,20 +38,28 @@ class ShieldhitDemo(Resource):
     """Class responsible for Shieldhit Demo running"""
 
     @staticmethod
-    def get():
+    def post():
```

Comment:
I would like to test it with curl,
add some description how to test it with curl (command line)

Reply:
Currently dummy converter ignores the content of json in requests body so it can contains anything. Example curl: `curl -i -X POST -H "Content-Type:application/json" -d "{\"Dummy\": \"Curl\" }" http://localhost:5000/sh/demo `